OnlineShop.Services.Basket.BusinessLayer/Models/Dto/OrderCreateDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OnlineShop.Services.Basket.BusinessLayer.Models.Dto
+{
+    public class OrderCreateDto
+    {
+        public string UserId { get; set; }
+
+        public IEnumerable<string> ProductIds { get; set; }
+
```

Comment:
in some of your dto classes you dont have spaces between properties and in others you have such spaces. come to a common approach